src/main/java/edu/kis/powp/jobs2d/features/DrawOnFreePanelFeature.java

```diff
@@ -0,0 +1,15 @@
+package edu.kis.powp.jobs2d.features;
+
+import edu.kis.powp.appbase.Application;
+import edu.kis.powp.jobs2d.drivers.DriverManager;
+
+public class DrawOnFreePanelFeature {
+
+	private static Application app;
```

Comment:
Is it really necessary here, to store Application? 
Moreover, it seems that dependency with the Application isn't really necessary.